tools/flowc/flowc_helpers.flow

```diff
@@ -533,7 +533,10 @@ getNameByLineColFromFile(line : int, col : int, file : string) -> string {
 
 getNameByLineCol(line : int, col : int, src : string) -> string {
 	resolver = makeLineResolver(src);
-	getNameByPos(findLineIndex(resolver, line, col), src);
+	getter = \c -> getNameByPos(findLineIndex(resolver, line, c), src);
+	name = getter(col);
+	// Fix for vscode behavior: it may give a first column after the last symbol if a word was selected.
+	if (name != "") name else getter(col - 1);
```

Comment:
You should add a guard against col < 0 to avoid infinite loop, hu?

Reply:
A guard is added